plan/physical_plan_builder.go

```diff
@@ -180,7 +180,7 @@ func (p *LogicalJoin) getIndexJoinByOuterIdx(outerIdx int) []PhysicalPlan {
 	if !ok || x.unionScanSchema != nil {
 		return nil
 	}
-	indices, includeTableScan := availableIndices(x.indexHints, x.tableInfo)
+	indices, includeTableScan, _ := availableIndices(x.indexHints, x.tableInfo)
```

Comment:
You should check error here but not in logical_plan_builder

Reply:
move hint index missing error check from logical_plan_builder to physical_plan_builder